src/fixtures/timelineAtom.tsx

```diff
@@ -6,78 +6,91 @@ const NewsEvents = [
         date: '1999',
         body:
```

Comment:
Probably worth adding some fixtures with a `toDate` and `toUnixDate` fields so that we capture them in chromatic

Reply:
The Julian Assange one should cover this already, but it looks like it only has those field on the first event. I wonder if something has changed if that was meant to be on all of them previously?